### ISSUE TITLE
Navigation arrows should be visible when they have keyboard focus

### ISSUE THREAD

**Issue by jvine** [sul-dlss/universalviewer#13]
As you tab through the viewer controls, the left and right arrows disappear as soon as you tab to them - I can't quite tell what triggers them to  appear and disappear, in fact, but they disappear when they have keyboard focus, and it should be the opposite... they should both a) be visible and b) have a visible focus state (which they do have on mouse hover).

**Comment by edsilv** [comment]
The left and right arrows, navigation window, and home, zoom etc buttons in OSD fade out after the configured time period.

You'll notice that when the mouse is over the navigation window (the small  box in the bottom right showing the whole image with your current zoom level as a white rectangle) the controls will not fade out. This is due to this:

https://github.com/UniversalViewer/universalviewer/blob/master/src/modules/uv-seadragoncenterpanel-module/SeadragonCenterPanel.ts#L285

Here's it's checking if the mouse is over the navigation window, if not `this.viewer.setControlsEnabled(false);` is called on OSD.

We should also check that the prev/next buttons are not moused over in a similar way, and whether they have focus before disabling the controls, e.g.

```
this.$prevButton.is( ":focus" )
```

**Comment by mejackreed** [comment]
Resolved upstream here: https://github.com/UniversalViewer/universalviewer/pull/488
